src/app/core/components/global-error/global-error.service.ts

```diff
@@ -12,8 +13,13 @@ import { GlobalError } from './global-error.interface';
 export class GlobalErrorService {
   private errors: Observable<GlobalError[]>;
 
-  constructor(private store: Store<State>) {
+  constructor(private store: Store<State>, private router: Router) {
     this.errors = this.store.pipe(select(globalErrorState));
+    this.router.events.subscribe((event)=> {
```

Comment:
Not super necessary, and I don't know if it's a better implementation, but the ngrx route store package dispatches actions when the user navigates. Maybe creating an effect which listens to these actions and clears the errors could be a cleaner/better implementation. Just a suggestion mostly
<img width="300" alt="ngrx" src="https://user-images.githubusercontent.com/104442665/198003615-a12e4aa9-4a89-423d-93f6-359ab6e517c4.PNG">

Reply:
Global errors reducer might be able to act on the `navigated` action and clear errors.

Reply:
I'll look into this, it's a good suggestion